tools/ansible/versioning-playbook.yaml

```diff
@@ -45,7 +45,7 @@
       - hacks/csv_copy_crd_descriptions.py
       - hacks/csv_copy_docs.py
       - hacks/csv_prep.py
-      - ../../../../docs/source/conf.py
+        # - ../../docs/source/conf.py
```

Comment:
Just for my understanding, what is this one used for, or what was it used for?

Reply:
It was used to  change the version  of the rtd